CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder.csproj

```diff
@@ -7,14 +7,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Communication.CallAutomation" Version="1.0.0-alpha.20230414.1" />
     <PackageReference Include="Azure.Communication.Identity" Version="1.2.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.3" />
-	<PackageReference Include="Azure.Communication.CallAutomation" Version="1.0.0-beta.1" />
-  </ItemGroup>
-
-  <ItemGroup>
```

Comment:
why you removed this, did you check if this code working after publish to azure?

Reply:
No Irfan. I was tested in Local, it is working fine.